services/bigid-azure-key-vault-integration-app/src/main/java/com/bigid/azurekeyvaultapp/service/impl/AzureKeyVaultTokenService.java

```diff
@@ -20,22 +19,30 @@ public class AzureKeyVaultTokenService implements KeyVaultTokenService {
     public AccessToken fetchAccessToken(ExecutionContext executionContext) {
         Map<String, String> globalParamsMap = ParamsMapUtils.getGlobalParamsMap(executionContext);
 
-        TokenCredential credential = globalParamsMap.get(GlobalParams.AUTHENTICATION_METHOD.getValue()).equals(GlobalParams.CLIENT_SECRET.getValue())
-                ? new ClientSecretCredentialBuilder()
-                .clientId(globalParamsMap.get(GlobalParams.CLIENT_ID.getValue()))
```

Comment:
do we validate that clientId + clientSecret values exist?

Reply:
> do we validate that clientId + clientSecret values exist?

sdk will throw IllegalArgumentException and we catch it later and return BAD_REQUEST